app/src/renderer/js/preferences.js

```diff
@@ -16,6 +16,22 @@ const settingsValues = app.kap.settings.getAll();
 // Observers that should be disposed when the window unloads
 const observersToDispose = [];
 
+function initVideoCodecSelector() {
```

Comment:
Can be an arrow function, `this` isn't used.

Reply:
Done